package.json

```diff
@@ -58,24 +39,23 @@
 		"@eslint/js": "9.23.0",
 		"@release-it/conventional-changelog": "10.0.0",
 		"@types/eslint-plugin-markdown": "2.0.2",
-		"@types/eslint__js": "8.42.3",
 		"@types/node": "22.13.11",
 		"@vitest/coverage-v8": "3.0.9",
-		"cspell": "8.13.1",
-		"eslint": "8.57.1",
+		"@vitest/eslint-plugin": "1.1.38",
+		"console-fail-test": "0.5.0",
```

Comment:
Related to https://github.com/JoshuaKGoldberg/console-fail-test/pull/1046/files#r2014657023, I'm pretty sure I'm going to have to make CTA able to opt out of `console-fail-test`...